tests/bpf2c_tests/elf_bpf.cpp

```diff
@@ -122,7 +122,7 @@ run_test_elf(const std::string& elf_file, _test_mode test_mode)
 
             REQUIRE(raw_result.size() == raw_output.size());
             for (size_t i = 0; i < raw_result.size(); i++) {
-                REQUIRE(raw_output[i] == raw_result[i]);
+                REQUIRE(((i >= 0) && (raw_output[i] == raw_result[i])));
```

Comment:
Why is this change needed?  size_t is unsigned so i>=0 should always be true, no?

Reply:
My bad. This was added to include the line number in the debug message when the test failed. Removed as it's no longer needed.

Reply:
My bad. This was added to include the line number in the debug message when the test failed. Removed as it's no longer needed.